Formula/mysql@5.6.rb

```diff
@@ -100,11 +100,8 @@ def install
     bin.install_symlink prefix/"scripts/mysql_install_db"
 
     # Fix up the control script and link into bin
-    inreplace "#{prefix}/support-files/mysql.server" do |s|
-      s.gsub!(/^(PATH=".*)(")/, "\\1:#{HOMEBREW_PREFIX}/bin\\2")
-      # pidof can be replaced with pgrep from proctools on Mountain Lion
-      s.gsub!(/pidof/, "pgrep") if MacOS.version >= :mountain_lion
```

Comment:
I believe it isn't as `support-files/CMakeLists.txt` now replaces `pidof` with `pgrep` on non-Linux systems, but can't test it out today since I am traveling.

> On non-Linux Unix systems, the mysql.server startup script used the Linux command pidof rather than pgrep. (Bug #13788154, Bug #64342)

from https://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-36.html